README.md

```diff
@@ -34,7 +34,22 @@ Define properties to override default bundle values
     <CFBundleIconFile>AppName.icns</CFBundleIconFile> <!-- Will be copied from output directory -->
     <NSPrincipalClass>NSApplication</NSPrincipalClass>
     <NSHighResolutionCapable>true</NSHighResolutionCapable>
+
+    <!-- Optional -->
+    <NSRequiresAquaSystemAppearance>true</NSRequiresAquaSystemAppearance>
 </PropertyGroup>
+
+<ItemGroup>
+    <!-- Optional URLTypes.Check TestBundle.csproj for a working example. -->
+    <CFBundleURLTypes Include="dummy"> <!-- The name of this file is irrelevant, it's a MSBuild requirement.-->
+        <CFBundleURLName>TestApp URL</CFBundleURLName>
+        <CFBundleURLSchemes>testappurl;testappurl://</CFBundleURLSchemes> <!-- Note the ";" separator-->
+    </CFBundleURLTypes>
+    <CFBundleURLTypes Include="dummy">
+        <CFBundleURLName>TestApp URL2</CFBundleURLName>
+        <CFBundleURLSchemes>test://;</CFBundleURLSchemes> <!-- Be sure to include a ";" separator as well when having 1 parameter, since CFBundleURLSchemes makes it mandatory to be an array -->
```

Comment:
it shouldnt be mandatory... you should handle that

Reply:
In CFBundleURLSchemes if you have 1 value, you need the structure 
`<array>
     <string>value</string>
 </array>` (https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102207).

But that's not the default behavior, and I want to make this agnostic to which parameter from the metadata are we writing to xml... 🤔

Reply:
but you can add the ";" if doesnt have... I mean this detail shouldnt be exposed to the developer

Reply:
Hmm ok, I see what you mean. Ty